### PR TITLE
[16.0][FIX] account_statement_import_sheet_file: Remove empty lines with spaces

### PR DESCRIPTION
**Pull request by mileo**
Some banks send a file with a space in the empty lines.

image

### CODE REVIEW

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]
Thanks for the PR! Please add a test case with sample file and I would consider stripping all whitespace-alike symbols with regex

**Review by alexey-pelykh — APPROVED** [review]
Nicely done!

**Comment by pedrobaeza** [comment]
Several things: 

- The commit message should include the module name. 
- Don't add empty lines inside a method. 
- Although previous test uses demo data, it's not a good practice, so please create the template for empty lines inside the test (although you can leave the demo one if you want).
- Should this really be an extra option or directly included for all the templates?

File: account_statement_import_sheet_file/tests/test_account_statement_import_sheet_file.py [hunk 1]
```diff
@@ -538,3 +538,51 @@ def test_amount_inverse_sign(self):
...
+                    "header_lines_skip_count": 1,
+                    "quotechar": '"',
+                    "float_thousands_sep": "dot",
+                    "reference_column": "MOVIMIENTO",
+                    "description_column": "DESCRIP",
+                    "amount_credit_column": "DEBE",
+                    "amount_debit_column": "HABER",
+                    "balance_column": "SALDO",
+                    "timestamp_format": "%d/%m/%Y",
+                    "timestamp_column": "FECHAMOVI",
```
Comment (rvalyi) [inline account_statement_import_sheet_file/tests/test_account_statement_import_sheet_file.py]: not so important: but probably better to put the values of the previous 7 dict items in English.

**Review by rvalyi — CHANGES_REQUESTED** [review]

**Comment by mileo** [comment]
Hi Guys, thank you for the review. I've added some unit tests to ensure that the conversion is working correctly for all cases.

**Comment by mileo** [comment]
> Several things:
> 
> * The commit message should include the module name.
> * Don't add empty lines inside a method.
> * Although previous test uses demo data, it's not a good practice, so please create the template for empty lines inside the test (although you can leave the demo one if you want).
> * Should this really be an extra option or directly included for all the templates?

All done... With these new tests, we can ensure that it will work for all cases, without any extra configuration.

**Review by rvalyi — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please don't use Decimal on tests. Just use regular floats.

**Comment by mileo** [comment]
> Please don't use Decimal on tests. Just use regular floats.

@pedrobaeza the code is base on decimal, the change to remove the Decimal is bigger.... Could you please review the second commit?

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge patch
